src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -33,6 +37,20 @@ public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHea
         switch (wrapper.getState()) {
             case CREATE_HEARING:
                 createHearing(wrapper);
+                HearingRequestPayload hearingRequestPayload = new HearingRequestPayload();
```

Comment:
I would create a new method for this, it's an already too complex method.

Reply:
Done